### PR TITLE
Update section

### PR DESCRIPTION
**Pull request by Ladeee**
changing the images from svg to png and working on the glows and other minor touches

### CODE REVIEW

File: __pages__/home/segments/partners/partners.module.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
   .partners {
     padding-top: 8rem /* 128px -> 8rem */;
-    background-image: url('/images/landing/helixes/partner.png');
+    background-image: url('/images/landing/helixes/partnerbg.png');
```
[the remark this answers is not in the document]
Comment (rajashree23) [reply __pages__/home/segments/partners/partners.module.scss:16]: change [this ](https://github.com/ChainWhiZ/chainwhiz-dapp-v2/blob/d9c8e4f4ad32bd3b3acd6f6665af1d84cbab53d2/__pages__/home/segments/partners/partners.module.scss#L6)background to this
    background: linear-gradient(180deg, rgb(17 12 29) 90%, rgba(16,12,26,1) 100%);

**Comment by rajashree23** [comment]
DESKTOP

1. Video Playback icon issue. - [https://onedrive.live.com/?authkey=%21ANmfPoEQJmesXd8&id=FE802492EDD642ED%213166&cid=FE802492EDD642ED](https://onedrive.live.com/?authkey=%21ANmfPoEQJmesXd8&id=FE802492EDD642ED%213166&cid=FE802492EDD642ED)
2. Ready to level up section and Class section BG blend problem
3. Our Partners- font-weight 500
4. Sneak Peek- font-weight to be changed
5. Got a minute- font weight to be changed
6. Your voice, Your opinions font weight 500
7. HOW WE WORK - ALL caps, font weight to 500
8. Our latest publication top right glow to be pushed to the right side, to get trimmed out into half.
9. A team that never stops, neon green glow to be increased radially, and white intensity to be increased.
10. Footer - Social media icons missing.
11. Font weight of footer option to be removed.
12. Backed by the very best- font weight to be changed
13. Create Solve Repeat, Neon green radius to be reduced on the right.
14. Create Solve Repeat, white glow missing underneath the illustration
15. White glow missing on the right in the footer section

**Review by rajashree23 — CHANGES_REQUESTED** [review]

**Comment by rajashree23** [comment]
MOBILE

1. Use this png for mobile screen homepage bg- https://onedrive.live.com/?authkey=%21ANmfPoEQJmesXd8&cid=FE802492EDD642ED&id=FE802492EDD642ED%213122&parId=FE802492EDD642ED%213103&o=OneUp
2. Play icon to be changed (mentioned in desktop comment)
3. Intensity of glow (neon) to be increased in the hero section.
4. Hero and Ready to level up BG to be  blended
5. Ready to level up para to be aligned with the below cards.
6. [https://github.com/ChainWhiZ/chainwhiz-dapp-v2/blob/main/__pages__/home/components/bounty/bounty.module.scss#L16](https://github.com/ChainWhiZ/chainwhiz-dapp-v2/blob/main/__pages__/home/components/bounty/bounty.module.scss#L16)- change the width to 23rem
7. WHen menu bar is opened. the page behind it seems to behave in a weird way. Please fix it.
8. MEnu bar needs correction,please connect on call
9. Our partners and View all bounties gapping is huge.
10. https://github.com/ChainWhiZ/chainwhiz-dapp-v2/blob/main/__pages__/home/components/partner/partner.module.scss#L12- width:23rem
11. neon green is distorted behind the  96% card
12. Your voice... text- increase font-size
13.  lot-->uppercase
14. how we work--uppercase
15. width of the latest publication cards to be increased. (23rem)
16. A team that never stops!---> A team that (one line) never stops(next line)
17. newsletter-> N caps
18. gapping to be reduced b/w wisescribe and team section
19. from wisescribe until footer bg color change--#100c1a
20. Enter you email input width to be increased(23rem)
21. footer- chainwhiz and social icon missing
22. footer menu- font-weight change
23. use this for footer bg- https://onedrive.live.com/?authkey=%21ANmfPoEQJmesXd8&cid=FE802492EDD642ED&id=FE802492EDD642ED%213168&parId=FE802492EDD642ED%213103&o=OneUp

**Comment by rajashree23** [comment]
Please update the logo to this
https://onedrive.live.com/?authkey=%21ANmfPoEQJmesXd8&cid=FE802492EDD642ED&id=FE802492EDD642ED%213169&parId=FE802492EDD642ED%213116&o=OneUp

**Comment by Xand6r** [comment]
@rajashree23 please continue the review in this pr(https://github.com/ChainWhiZ/chainwhiz-dapp-v2/pull/53)
I just noticed this has the wrong base branch so i'm gonna close it down after you acknowledge seeing this.

**Comment by rajashree23** [comment]
> @rajashree23 please continue the review in this pr(#53) I just noticed this has the wrong base branch so i'm gonna close it down after you acknowledge seeing this.

are the review comments addressed [here](https://github.com/ChainWhiZ/chainwhiz-dapp-v2/pull/50) or this [PR](https://github.com/ChainWhiZ/chainwhiz-dapp-v2/pull/53)?

Please keep it open cause I want to see the changes that are made wrt my comments and the CSS changes requested by me

**Comment by Xand6r** [comment]
sure, all the changes can be found in the other PR @rajashree23

**Comment by rajashree23** [comment]
not required

**Pull request closed without merging**